### ISSUE TITLE
`CssBorder::Border` not working as I would expect

### ISSUE THREAD

**Issue by eopb** [rebo/seed_style_preview#6]
The code:

```rust
.border(CssBorder::Border(
    px(6).into(),
    CssBorderStyle::Solid,
    "green".into()
))
```

seems to result in the CSS rules:

```css
border-style: solid;
color: green;
```

I would expect:
```css
border: 6px solid green;
```

**Comment by rebo** [comment]
to confirm does it drop the width complete i.e. 6px?

And also confirm you would like a single "border:  xxxxxx" property ? rather than 3 separate ones?

**Comment by eopb** [comment]
As far as I can tell it drops the 6px entirely. I can't find any mention of it in the CSS inspector.

I would expect a single border: xxxxxx" rule, yes.

**Comment by rebo** [comment]
Closing as fixed in 00bfb29 .
